src/main/java/com/liferay/damascus/cli/common/CaseUtil.java

```diff
@@ -0,0 +1,63 @@
+package com.liferay.damascus.cli.common;
+
+/**
+ * Case formats conversion utility
+ * 
+ * @author Sébastien Le Marchand
+ */
+public class CaseUtil {
+	
+	protected CaseUtil() {
+		
+	}
+	
+    /**
+     * Get Instance
+     *
+     * @return this instance
+     */
+    public static CaseUtil getInstance() {
```

Comment:
Guava has some limitations with uppercase characters sequences.

For example, converting from camel case to dash case the string "UnknownURL":
* with Guava: "unknown-u-r-l"
* with the custom class: "unknown-url"